sites/docs/content/get-started.md

```diff
@@ -5,11 +5,45 @@ tagline: Guide
 ---
 
 <script>
-	import { Callout } from '$lib/components'
+	import { Steps, Step } from "$lib/components";
 </script>
 
-<Callout type="warning">
+<Steps>
+<Step>Install the package</Step>
 
-The documentation is still a work in progress. Please refer to the [GitHub README](https://github.com/svecosystem/mode-watcher) for more information.
+You can install the project via `npm` or `pnpm`.
 
-</Callout>
+```bash
+pnpm add mode-watcher
+```
+
+<Step>Add the ModeWatcher component</Step>
+
+Add the `<ModeWatcher />` component to your root `+layout.svelte` file.
+
+```svelte {2,5}#add
```

Comment:
not sure about indentation here... are we supposed to indent after using `<Steps>`?

Reply:
if I try to indent it's reverted by prettier it seems

Reply:
Indentations in markdown are treated as code blocks if there is a line between the component and the plaintext!